apps/token-manager/app/src/components/AppLayout.js

```diff
@@ -21,18 +22,24 @@ const AppLayout = ({
           appBar={
             <AppBar>
               <AppBarContainer
-                style={{ padding: below('medium') ? '0' : '0 30px' }}
+                style={{ padding: below('medium') ? '0' : '0 30px 0 10px' }}
               >
                 <Title>
-                  {below('medium') && <MenuButton onClick={onMenuOpen} />}
-                  <TitleLabel>{title}</TitleLabel>
+                  {displayMenuButton && <MenuButton onClick={requestMenu} />}
```

Comment:
Could we clean up the `MenuButton` implementations a bit, since the icon is now exposed in aragonUI?